docs/videos/index.html

```diff
@@ -96,26 +96,33 @@ <h4>Short Introduction to Web of Things
     <h3>Interviews, Meetings, Conferences</h3>
     <br />
     <div class="row">
+        <div class="col-md-6">
+            <a target="_blank" href="https://www.youtube.com/watch?v=PuhNX0QdvYI&list=PLDXXQJiSjPKHyo7D9xDLJELFQw5GHKQYJ&index=3">
+                <img class="img-responsive" style="min-width: 180px; min-height: 245px; width: 100%; border-width: thin; border-style: solid;"
+                    src="{{'/images/video-thumbnail-riot-2022.png' | absolute_url }}" alt="WoT Video" />
+                <h4>RIOT Summit 2022, Keynote: "Web of Things - What is it all about"</h4>
+                <!-- 2021-11-05 -->
```

Comment:
I guess this comment about the date is not correct

Reply:
```suggestion
                <!-- 2022-09-05 -->
```

Reply:
fixed :)